src/main/cpp/t04_swap.cpp

```diff
@@ -25,6 +25,26 @@
 using namespace std;
 
 int t04_swap() {
-    
+    int n, m, i, j, swap;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'swap' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4436267)